canopeum_frontend/src/components/analytics/AnalyticsSiteHeader.tsx

```diff
@@ -94,15 +95,15 @@ const AnalyticsSiteHeader = ({ siteSummary }: Props) => {
           />
         </div>
 
-        <div className='mt-4'>
-          <div
-            className='d-flex align-items-flex-end fw-bold overflow-x-auto'
-            style={{ maxWidth: '800px' }}
-          >
-            <span className='material-symbols-outlined'>group</span>
-            <span className='ms-1 me-2'>{translate('analyticsSite.sponsors')}:</span>
+        <div
+          className='d-flex align-items-center fw-bold overflow-x-auto pb-2 mt-4'
+          style={{ maxWidth: '800px' }}
```

Comment:
Magic number spotted ;)
Why 800 specifically? Does that represent anything? A specific breakpoint? Based on some page-specific requirements ?